doc/2/api/controllers/document/m-exists/index.md

```diff
@@ -0,0 +1,126 @@
+---
+code: true
+type: page
+title: mExists
+---
+
+# mExists
+
+<SinceBadge version="auto-version"/>
+
+Check if multiple documents exists.
+
+::: info
+The number of documents that can be fetched by a single request is limited by the `documentsFetchCount` server configuration (see the [Configuring Kuzzle](/core/2/guides/advanced/configuration) guide).
+:::
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_mExists
+Method: POST
+Body:
+```
+
+```js
+{
+  "ids": ["<documentId>", "<anotherDocumentId>"]
+}
+```
+
+You can also access this route with the `GET` verb:
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_mExists?ids=documentId,anotherDocumentId
+Method: GET
+```
+
+### Other protocols
+
+```js
+{
+  "index": "<index>",
+  "collection": "<collection>",
+  "controller": "document",
+  "action": "mExists",
+  "body": {
+    "ids": ["<documentId>", "<anotherDocumentId>"]
+  }
+}
+```
+
+### Kourou
+
+```bash
+kourou document:mExists <index> <collection> <body>
+```
+
+---
+
+## Arguments
+
+- `collection`: collection name
+- `index`: index name
+
+### Optional:
+
+- `strict`: if set, an error will occur if any of the documents does not exists
+
+---
+
+## Body properties
+
+- `ids`: an array of document identifiers to fetch
+
+---
+
+## Response
+
+Returns an object containing 2 arrays: `successes` and `errors`
+
+The `successes` array contain the list of retrieved documents.
```

Comment:
The documentation is incorrect since you are not returning the entire document.  

Also I think we should only returns 2 arrays, one containing existing document and the other one containing missing documents

Reply:
@matissepage